src/content/app/species-selector/SpeciesSelector.tsx

```diff
@@ -30,13 +31,14 @@ import SpeciesSelectorGeneSearchView from './views/species-selector-gene-search-
 import styles from './SpeciesSelector.module.css';
 
 const SpeciesSelector = () => {
+  const { pathname } = useLocation();
+  const [searchParams] = useSearchParams();
   const modalView = useAppSelector(getSpeciesSelectorModalView);
   const dispatch = useAppDispatch();
 
-  const shouldShowResultsView = [
-    'species-search',
-    'popular-species-genomes'
-  ].includes(modalView || '');
+  const shouldShowResultsView =
+    pathname.includes('/search') &&
```

Comment:
Shouldn't this (the `/search` part) be done via react-router's react components? See EntityViewer as an example.

Reply:
✅